packages/dataviews/src/style.scss

```diff
@@ -3,6 +3,7 @@
 	height: calc(100% - #{$grid-unit-40} * 2);
```

Comment:
There is [an issue](https://github.com/storybookjs/storybook/issues/8112) with storybook iframe's height that results in `sticky` headers not to work properly. I'm not sure if this should be treated as storybook issue or it indicates that we should handle somehow such a use case. --cc @ciampo

Reply:
How can we handle it in a way that works everywhere (regardless of how dataviews is used)

Reply:
You can test the story outside of the Storybook canvas iframe ([example](https://wordpress.github.io/gutenberg/iframe.html?args=&id=dataviews-experimental-dataviews--default&viewMode=story)) to understand if the iframe could be the issue here